.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -124,8 +136,18 @@ dynamodb_DeleteTable:
             - description:
               snippet_tags:
                 - dynamodb.python.codeexample.MoviesDeleteTable
+    Rust:
+      versions:
+        - sdk_version: 1
+          github: .rust_alpha/dynamodb
+          excerpts:
+            - description:
+              snippet_tags:
+                - dynamodb.rust.crud-remove_table
+                - dynamodb.rust.delete-table
+                - dynamodb.rust.movies-delete_table

```

Comment:
Some of these snippets seem to contain the same code. Add descriptions or remove dups?

Reply:
I've removed the duplicates.